docs/source/deployment/lambda.md

```diff
@@ -88,6 +309,10 @@ functions:
         path: /
         method: get
         cors: true
+    - http:
```

Comment:
are you sure this is necessary? i think this is what `cors: true` does? or if `cors: true` isn't adequate and we need Apollo Server to handle the OPTIONS itself, then we probably should remove the `cors: true`s?

Reply:
Tested this out and you're right. I probably made a bad assumption on the serverless framework here after seeing it required in the AWS Lambda findings.